app/api/routers/breeze_buddy/widget/handlers.py

```
@@ -323,6 +327,100 @@ async def send_widget_message_handler(
+            data,
+            audio.content_type,
+            provider=provider,
+            language=language,
+            filename=audio.filename or "audio.webm",
+        )
+    except TranscriptionError as e:
+        logger.warning("widget transcribe failed (session={}): {}", session_id, e)
+        raise HTTPException(
+            status_code=status.HTTP_502_BAD_GATEWAY,
+            detail="Transcription failed",
+        )
```

Ruff (0.15.17)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)